src/main/flight/pid.c

```diff
@@ -256,6 +262,21 @@ void resetPidProfile(pidProfile_t *pidProfile)
     );
 }
 
+static bool isTpaActive(tpaMode_e tpaMode, term_e term) {
+    switch (tpaMode) {
+    case TPA_MODE_PD:
+        return (term == TERM_P || term == TERM_D);
+    case TPA_MODE_D:
+        return (term == TERM_D);
+#ifdef USE_WING
+    case TPA_MODE_PDS:
+        return (term == TERM_P || term == TERM_D || term == TERM_S);
```

Comment:
```suggestion
        return term == TERM_P || term == TERM_D;
    case TPA_MODE_D:
        return term == TERM_D;
#ifdef USE_WING
    case TPA_MODE_PDS:
        return term == TERM_P || term == TERM_D || term == TERM_S;
```

Reply:
cmon, its nicer with brackets :)

Reply:
https://style.mla.org/how-to-use-parentheses/

Reply:
Sorry reading code is more easy without if not necessary. Parenthesis can make group conditions.

Reply:
@ledvinap would agree [ or not :) ]

Reply:
BF style guide recommends avoiding all unnecessary parentheses.

But I have no problem with them, they are easy to ignore.

Reply:
ok, ok, removed :)